Performance/perf_runner_swift.sh

```diff
@@ -28,6 +28,9 @@ function print_swift_set_field() {
       echo "          message.field$num.append(Data(repeating:$((num)), count: 20))"
       echo "        }"
       ;;
+    repeated\ bool)
+      echo "        message.field$num = [true, false, true, false, true, false, true, false]"
```

Comment:
I would complain that this ignores the value of `repeatedCount` if I actually made smart use of it anywhere. :)